common/defaults.go

```diff
@@ -178,6 +178,9 @@ const (
 	// ArgoCDKeycloakImage is the default Keycloak Image used for the non-openshift platforms when not specified.
 	ArgoCDKeycloakImage = "quay.io/keycloak/keycloak"
 
+	// Default Label Selector is used when the Label Selector is not mentioned/empty.
```

Comment:
Probably you can mention that, `by default all ArgoCD instances would be reconciled`.

Reply:
Also the go doc comment should start with the variable name i.e `ArgoCDDefaultLabelSelector`